src/components/menu-item/menu-item.styles.ts

```diff
@@ -20,8 +20,8 @@ export default css`
     font-weight: var(--sl-font-weight-normal);
     line-height: var(--sl-line-height-normal);
     letter-spacing: var(--sl-letter-spacing-normal);
-    color: var(--sl-color-neutral-700);
-    padding: var(--sl-spacing-2x-small) var(--sl-spacing-2x-small);
+    color: var(--ts-color-text-default);
```

Comment:
I have reservations about using the `ts--` variables directly in the Shoelace stylesheets, because then we're kind of creating a circular dependency. As I understand it, we shouldn't need to use those values directly, because we're overriding the Shoelace values with their corresponding `ts` values already. Open to discussion on that.

Reply:
Ah, I see -- so we should use `--sl-color-neutral-900` here, you mean? I was thinking this makes the code more readable for us, since we know right away what `ts-color-text-default` is. But I'm fine to update all these to remove the `--ts` variable and replace with the `--sl` equivalent. 

There are a few places where we had to create our own hard-coded `--ts` token rather than overriding an existing `--sl` token because the value we wanted kinda fell between two `--sl` tokens that we wanted to keep. In those cases, do you think it's okay to reference the `--ts` token directly? 

Also -- okay with you if I just keep that in mind as the new approach & update these instances in the new branch that I'm working on?

Reply:
Yeah, that's fine to just update in the other branch. I think if we're creating new values, we might want to just create those as `--sl` values as well, so we have consistency in our approach across the board.

Reply:
ah, okay. we have several of these custom `-ts` values so i think i will pull this work of updating this into a separate ticket, then? but what you are saying makes sense to me. agree that we should be consistent!

Reply:
Yep, that's fine.